specfile/changelog.py

```diff
@@ -194,6 +194,77 @@ def style(self) -> ChangelogStyle:
             return ChangelogStyle.openSUSE
         return ChangelogStyle.standard
 
+    @property
+    def author(self) -> Optional[str]:
+        """The author of this changelog entry including their email."""
+        if self.style == ChangelogStyle.openSUSE:
+            # openSUSE format: "OPENSUSE_CHANGELOG_SEPARATOR\n$DATE - $AUTHOR <$EMAIL>"
+            # Extract everything after " - " from the actual header line (second line)
+            header_lines = self.header.split("\n")
+            if len(header_lines) > 1:
+                if len(date_author := header_lines[1].split("-", maxsplit=1)) > 1:
+                    return date_author[1].strip()
+        else:
+            # Standard format: "* $DATE $AUTHOR <$EMAIL> [- $EVR]"
+            header_without_asterisk = self.header.removeprefix("* ")
+
+            # Find the end of the date portion by looking for year (4 digits followed by space)
+            year_match = re.search(r"\d{4}\s+", header_without_asterisk)
+            if year_match:
+                author_and_evr = header_without_asterisk[year_match.end() :]
+
+                if not self.evr:
+                    return author_and_evr
+
+                if (gt_ind := author_and_evr.rfind(">")) > 0:
+                    return author_and_evr[: gt_ind + 1].strip()
+
+        return None
+
+    @property
+    def timestamp(self) -> datetime.datetime:
```

Comment:
Could this return `datetime.date` for non-extended timestamps, to mirror the behavior of `add_changelog_entry()`?